### PR TITLE
WQXR-209 Clear host image metadata

### PR DESCRIPTION
**Pull request by bkammin**

### CODE REVIEW

File: app/routes/application.js
Comment (walsh9) [inline app/routes/application.js:87]: does this end up any different than just calling
`stream.set('hostImageTemplateUrl', hostImageTemplateUrl);`
without the if block?

i guess `stream.set('hostImageTemplateUrl', hostImageTemplateUrl || undefined);` would be more exact if we want to be specific about handling other falsey values....

File: app/routes/application.js
Comment (walsh9) [inline app/routes/application.js:94]: same as above

File: app/routes/application.js
Comment (walsh9) [inline app/routes/application.js:118]: something feels weird to me about this, like i want to say we could just call clearShowMetadata at the top every time and then set on top that if we need to instead of another conditional,  but the async findRecord call here means that this wouldn't actually be the same since there could be a delay, so we do need to do it how it is here. I think this is good.

**Review by walsh9 — APPROVED** [review]
Have some questions/suggestions but i'm going to approve and leave it up to you if you want to change anything

File: app/routes/application.js
Comment (bkammin) [reply app/routes/application.js]: great point. done!

**Pull request merged**
